coalib/bears/Bear.py

```diff
@@ -130,6 +130,12 @@ class Bear(Printer, LogPrinterMixin):
     >>> class SomeBear(Bear): pass
     >>> SomeBear.source_location
     '...Bear.py'
+
+    Every lint bear makes use of an executable tool for its operations.
+    The SEE_MORE attribute provides a link to the main page of the tool:
+
+    >>> class SomeBear(Bear):
```

Comment:
let's use a real bear, so that the doctests itself show what `SEE_MORE` means! :smile: